Console/Makefile

```diff
@@ -0,0 +1,26 @@
+# path to RETRO68
+RETRO68=../../Retro68-build/toolchain
```

Comment:
Probably shouldn't include the `Makefile`s, that is bound to confuse people.

Reply:
I needed them, as I don't know how to use CMake yet.
Please remove them if you think they are not appropriate here!

Reply:
I agree, I needed them as I don't know how to use CMake yet :-|

Reply:
I'm specifically afraid that people who are used to Makefiles and unfamiliar with CMake will be confused by the presence of Makefiles that aren't actually invoked by the build system.